nbconvert/exporters/markdown.py

```diff
@@ -30,6 +30,20 @@ def _raw_mimetypes_default(self):
 
     @property
     def default_config(self):
-        c = Config({'ExtractOutputPreprocessor':{'enabled':True}})
+        c = Config({
+            'ExtractOutputPreprocessor':{'enabled':True},
+            'NbConvertBase': {
+                'display_data_priority' : ['application/javascript',
```

Comment:
Probably want to exclude javascript here, as that won't usually work in markdown. Otherwise, 👍